oci/layout/oci_transport.go

```diff
@@ -150,7 +152,7 @@ func (ref ociReference) DockerReference() reference.Named {
 // not required/guaranteed that it will be a valid input to Transport().ParseReference().
 // Returns "" if configuration identities for these references are not supported.
 func (ref ociReference) PolicyConfigurationIdentity() string {
-	return fmt.Sprintf("%s:%s", ref.resolvedDir, ref.tag)
+	return fmt.Sprintf("%s:%s", ref.resolvedDir, ref.image)
```

Comment:
OK; then `ValidatePolicyConfigurationScope` should also accept only directory values, not dir`:`image (but enforce that the directories don’t contain `:`, so that they can actually match `resolvedDir` values) and `PolicyConfigurationNamespaces` should, to be strictly correct, return only parent directories of `resolvedDir`, not `resolvedDir` itself (though, admittedly, technically, returning `resolvedDir` here as well does not actually hurt anything IIRC).